### DISCUSSION TITLE
How to enable Wiegand RFID

### DISCUSSION THREAD

**Discussion by ChrisVonFrosta** [arendst/Tasmota#11317] [Q&A] [answered]
How could i enable/activate the Wiegand rfid interface?
Do i need to compile my own binary?

thx!

**Comment by ascillato** [comment]
Yes

**Comment by ChrisVonFrosta** [comment, marked as answer]
thx but uff  ;) 
where i have to include this (uncomment which line)?
i have the source open in Visial Studio.

**Reply by ChrisVonFrosta** [reply]
my_user_config.h ?

**Reply by barbudor** [reply]
Please read https://tasmota.github.io/docs/Compile-your-build/

You should not change `my_user_compile` but place all of your `#define` and `#undef` in `user_config_override.h`

You can see all possible options that are listed in `my_user_config.h` but changing it will prevent it to be upgraded with newer versions. while `user_config_override.h` remains personnal.
The easiest is to copy lines of interrest from the 1st to the 2nd and removing the comment there.

**Reply by ChrisVonFrosta** [reply]
thx for yourpoint to user_config_override.h!

i am able to compile the right project (sensor).
i also enable the wiegand interface.

thx!
